src/layouts/components/Homepage/HeroBody.tsx

```diff
@@ -71,7 +71,7 @@ export const HeroBody = ({
   return (
```

Comment:
will be good to have a before ss also to compare while reviewing

Reply:
added